algo/src/test/kotlin/com/hartwig/actin/algo/evaluation/tumor/MeetsSpecificCriteriaRegardingMetastasesTest.kt

```diff
@@ -0,0 +1,40 @@
+package com.hartwig.actin.algo.evaluation.tumor
+
+import com.hartwig.actin.algo.evaluation.EvaluationAssert
+import com.hartwig.actin.algo.evaluation.EvaluationFactory
+import com.hartwig.actin.datamodel.algo.EvaluationResult
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Test
+
+class MeetsSpecificCriteriaRegardingMetastasesTest {
+
+    private val patientRecord = TumorTestFactory.withTumorStage(null)
+
+    @Test
+    fun `Should fail when no metastatic cancer`() {
+        val alwaysFailsMetastaticCancerEvaluation = mockk<HasMetastaticCancer> {
+            every { evaluate(any()) } returns EvaluationFactory.fail("no metastatic cancer")
+        }
+        val function = MeetsSpecificCriteriaRegardingMetastases(alwaysFailsMetastaticCancerEvaluation)
+        EvaluationAssert.assertEvaluation(EvaluationResult.FAIL, function.evaluate(patientRecord))
+    }
+
+    @Test
+    fun `Should be undetermined when tumor stage unknown`() {
```

Comment:
"tumor stage unknown" refers to the implementation details of `HasMetastaticCancer`, which isn't being tested here. I would rename this to something like:
```suggestion
    fun `Should be undetermined when unknown if patient has metastatic cancer`() {
```